src/components/05_pages/Permissions/Permissions.js

```diff
@@ -221,30 +224,36 @@ const Permissions = class Permissions extends Component {
       return <Loading />;
     }
     return (
-      <Fragment>
-        <input
-          type="text"
-          placeholder="Filter by name, description or module"
-          onChange={this.handleKeyPress}
-          onKeyDown={this.handleKeyPress}
-        />
-        <button
-          key="button-save-roles"
-          onClick={this.saveRoles}
-          disabled={
-            !this.state.changedRoles.length ||
-            (this.state.working && this.state.changedRoles.length)
-          }
-        >
-          Save
-        </button>
+      <StickyContainer>
+        <Sticky>
+          {({ style }) => (
+            <div style={style} className={styles.stickyBar}>
+              <input
+                type="text"
+                className={styles.searchInput}
+                placeholder="Filter by name, description or module"
+                onChange={this.handleKeyPress}
+                onKeyDown={this.handleKeyPress}
+              />
+              <button
+                key="button-save-roles"
+                onClick={this.saveRoles}
+                className={styles.saveButton}
+                disabled={
+                  !this.state.changedRoles.length ||
+                  (this.state.working && this.state.changedRoles.length)
+                }
+              >
+                Save
+              </button>
+            </div>
+          )}
+        </Sticky>
         <Table>
           <THead
```

Comment:
A future idea: It would be nice to get sticky headers for the table header too. I tried to get it working, but another ```<Sticky>``` header, even with an additional 80 pixels of offset didn't work.